packages/ledger/src/wallet/utils.ts

```diff
@@ -0,0 +1,77 @@
+import Transport from '@ledgerhq/hw-transport-node-hid';
+import {
+  AminoSignResponse,
+  encodeSecp256k1Signature,
+  OfflineAminoSigner,
+  serializeSignDoc,
+  StdSignDoc
+} from '@cosmjs/amino';
+import {fromHex } from '@cosmjs/encoding';
+import { AccountData } from '@cosmjs/proto-signing';
+import { Secp256k1Signature } from '@cosmjs/crypto';
+import { SeiApp } from '@zondax/ledger-sei';
+
+/**
+ *  A signer implementation that uses a Ledger device to sign transactions
+ */
+export class LedgerOfflineAminoSigner implements OfflineAminoSigner {
```

Comment:
nice, this is great!  should we name this `SeiLedgerOfflineAminoSigner` so it's more specific?